### ISSUE TITLE
wk2 aug_epic: Story 1 standardize research workflow in teampsd manual

### ISSUE THREAD

**Issue by ritahitching** [lzim/teampsd#1362]
*Create a .md and YAML Header Style guide for Manuscripts*** 

.md and YAML Header Style guides may also be helpful in OSF for specific types of Scientific Journals

## This guide would define:
- [ ] YAML header of the .md file
- [ ] Heading levels allowed
- [ ] Tables and Figures
- [ ] .csl citation language used for the manuscript
- [ ] Add to document_tracker disseminate_scientist_va column 
- [ ] Start w/APA...
- [ ] Look and what other publishers we are targeting for our manuscripts.

Use march wk4 team_times for this.

**Comment by lzim** [comment]
Great job outlining this @ritahitching ! 👏

**Comment by ritahitching** [comment]
Cross referencing
- [ ] Issue #1364 Due 5/15 task Team PSD Bookdown Manual: Define header level standards & checklist for review of online resources
- [ ] Issue #1192 Dur 5/15 Team PSD Manual 

FYI @lijenn @staceypark @anthonycpichardo @lzim

**Comment by ritahitching** [comment]
@ritahitching  Existing resources:
- [ ] [csl_styles](https://github.com/lzim/teampsd/edit/rita_2020_04_27_issue_1364/pubs_workgroup/csl_styles/README.md)

**Comment by ritahitching** [comment]
@lijenn this is issue is blocked by the #1192 and the MVP process.
I recommend we keep this issue open.

**Comment by lzim** [comment]
Hi @lijenn and @staceypark Is this still open?

**Comment by staceypark** [comment]
@lijenn ?

**Comment by lijenn** [comment]
@lzim @staceypark I don't believe this card was started on yet so it is an open card. 

But the task seems to standardize or set up a guide on how we should set up our manuscripts (either on OSF or RStudio?) based on the publishing journal and standard practices on setting up that. 

I recall Lindsey posting [this](https://github.com/andrewheiss/ngo-crackdowns-philanthropy/blob/9e7e78b9d3c5e1849b5be7faa6d8eecf32063a40/manuscript/Makefile) and we could use that file and other docs in that manuscript folder as a starter reference to see how we could approach this. Though, this may be slightly dependent on the journals we're in interested in?

**Comment by staceypark** [comment]
@lijenn @lzim I don't think we need this for the current manuscripts we're working on getting out the door so I'm going to move it to December.

**Comment by staceypark** [comment]
@anthonycpichardo @lzim 
as part of the Story 1 goal to turn to OSF project workflow in March, I propose we move it to then.
however, I think that's specific to my portion of this issue but not Anthony's.

Anthony, how are you conceptualizing your piece for this and where should we align with our other workflows?

**Comment by staceypark** [comment]
@dkngenda I'm adding you to this issue as the new Story 1 lead

**Comment by staceypark** [comment]
3/31 support wkgps:
to be discussed further during 4/5 story 1-4 grants alignment workshop

**Comment by staceypark** [comment]
separated out story 4 into a separate issue #1902 and removing myself from this one.

**Comment by lzim** [comment]
**Discussed at team_time 8/10/21**

### Compliance - Primarily focused on Data (not code)
1. What we required to comply with in terms of **VHA policy/guidance**, e.g., GitHub.com vs. GitHub.ec.va.gov
2. Where we **report data** is stored (RedCap, Team PSD Folder, VINCI) - Always need to stay consistent w/what we report.

### Priorities:** 

**Provider Survey - Monthly Update #218** 
- [ ] Redcap to R API 
- [ ] GitHub Branches
- [ ] R Notebook -> Code Chunk for Provider Survey 
   - [ ] data import 
   - [ ] clean 
   - [ ] transform 
   - [ ] descriptive

**Facilitator Fidelity - Monthly Update #219** 
- [ ] Redcap to R API 
- [ ] GitHub Branches
- [ ] R Notebook -> Code Chunk for Monthly Update Issue # Survey 
   - [ ] data import 
   - [ ] clean 
   - [ ] transform 
   - [ ] descriptive

**R21 & Special Issue Manuscripts**
